gcloud-java-examples/src/main/java/com/google/gcloud/examples/dns/DnsExample.java

```diff
@@ -67,7 +67,6 @@
 public class DnsExample {
 
   private static final Map<String, DnsAction> ACTIONS = new HashMap<>();
-  private static final DateFormat FORMATTER = new SimpleDateFormat("YYYY-MM-dd HH:mm:ss");
```

Comment:
You could have kept it as static if you were using the Joda class `DateTimeFormat.forPattern("YYYY-MM-dd HH:mm:ss")`